### PR TITLE
add vmm_active_uplinks_check + tests + doc

### PR DESCRIPTION
**Pull request by monrog2**

### CODE REVIEW

**Comment by monrog2** [comment]
test output:
```
[Check  1/1] fvUplinkOrderCont with blank active uplinks definition...                                            FAIL - OUTAGE WARNING!!
  Tenant  Application Profile  Application EPG  VMM Domain
  ------  -------------------  ---------------  ----------
  test1   ap1                  epg1             test-dvs
  test1   ap2                  epg2             test-dvs
  test1   ap3                  epg3             test-dvs

  Recommended Action: Identify Active Uplinks and apply this to the VMM domain association of each EPG
  Reference Document: https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations#access-untagged-port-config


[Check  1/1] fvUplinkOrderCont with blank active uplinks definition...                                                               PASS
```
reached out to submitter for doc content review.

File: aci-preupgrade-validation-script.py [hunk 1]
```diff
@@ -2906,6 +2906,36 @@ def access_untagged_check(index, total_checks, **kwargs):
...
 
 
+def vmm_active_uplinks_check(index, total_checks, **kwargs):
+    title = 'fvUplinkOrderCont with blank active uplinks definition'
+    result = PASS
+    msg = ''
+    headers = ["Tenant", "Application Profile", "Application EPG", "VMM Domain"]
+    data = []
+    recommended_action = 'Identify Active Uplinks and apply this to the VMM domain association of each EPG'
+    doc_url = 'https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations#access-untagged-port-config'
```
Comment (monrog2) [inline aci-preupgrade-validation-script.py:2916]: Need to update this URL with the right section

**Comment by monrog2** [comment]
integration tests look good.

pytest output:
```
[Check  1/1] fvUplinkOrderCont with blank active uplinks definition...                                            FAIL - OUTAGE WARNING!!
  Tenant  Application Profile  Application EPG  VMM Domain
  ------  -------------------  ---------------  ----------
  test1   ap1                  epg1             test-dvs
  test1   ap2                  epg2             test-dvs
  test1   ap3                  epg3             test-dvs

  Recommended Action: Identify Active Uplinks and apply this to the VMM domain association of each EPG
  Reference Document: https://datacenter.github.io/ACI-Pre-Upgrade-Validation-Script/validations#vmm-uplink-container-with-empty-actives


[Check  1/1] fvUplinkOrderCont with blank active uplinks definition...                                                               PASS
[Check  1/1] fvUplinkOrderCont with blank active uplinks definition... cversion does not have class fvUplinkOrderCont                 N/A

File: aci-preupgrade-validation-script.py [hunk 2]
```diff
@@ -2978,6 +2987,40 @@ def eecdh_cipher_check(index, total_checks, cversion, **kwargs):
...
+    uplink_api += '?query-target-filter=eq(fvUplinkOrderCont.active,"")'
+    vmm_epg_regex=r"uni/tn-(?P<tenant>[^/]+)/ap-(?P<ap>[^/]+)/epg-(?P<epg>[^/]+)/rsdomAtt-\[uni/vmmp-.+/dom-(?P<dom>.+)\]"
+
+    try:
+        affected_uplinks = icurl('class', uplink_api)
+    except OldVerClassNotFound:
+        # Pre 4.x did not have this class
+        msg = 'cversion does not have class fvUplinkOrderCont'
+        result = NA
+        print_result(title, result, msg, headers, data, recommended_action=recommended_action, doc_url=doc_url)
```
Comment (takishida) [inline aci-preupgrade-validation-script.py:3010]:
```suggestion
        print_result(title, result, msg)
```

File: aci-preupgrade-validation-script.py [hunk 3]
```diff
@@ -2978,6 +2987,40 @@ def eecdh_cipher_check(index, total_checks, cversion, **kwargs):
...
+        return result
+
+
```
Comment (takishida) [inline aci-preupgrade-validation-script.py:3013]:
```suggestion
```

**Review by takishida — CHANGES_REQUESTED** [review]

**Review by takishida — APPROVED** [review]

**Pull request merged**